ai2thor/server.py

```diff
@@ -122,6 +122,9 @@ def __init__(self, metadata):
 
         self.events = [self]  # Ensure we have a similar API to MultiAgentEvent
 
+    def __bool__(self):
```

Comment:
You should add a __bool__ method into MultiAgentEvent that delegates to self._active_event.__bool__

Reply:
added: https://github.com/allenai/ai2thor/pull/623/commits/2cb3ca45f0721b13003ff3f13cdc7c5b7cbc2877